test_scripts/API/PerformAudioPassThru/017_ATF_N_PerformAudioPassThru_INVALID_DATA_All_Params_audioPassThruIcon_invalid_ImageType.lua

```diff
@@ -0,0 +1,110 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PerformAudioPassThru] SDL must transfer request to HMI in case of valid "audioPassThruIcon" param
+-- [GeneralResultCodes] INVALID_DATA wrong type
+-- [Mobile API] PerformAudioPassThru request/response
+-- [HMI_API] [MOBILE_API] The "audioPassThruIcon" param at "ImageFieldName" struct
+--
+-- Description:
+-- In case that the request comes with
+-- wrong type (empty) parameter(including parameters of the structures)
+-- SDL must respond with resultCode "INVALID_DATA" and success:"false"
+--
+-- 1. Used preconditions
+-- 1.1. Request is sent with audioPassThruIcon image structure with invalid ImageType parameter
+-- and valid value parameter, all other params used in PerformAudioPassThru_request
```

Comment:
@maria-mih, will be better to specify which invalid types will be checked: empty and integer.